lib/node_modules/@stdlib/math/base/assert/is-prime/src/main.c

```diff
@@ -20,6 +20,7 @@
 #include "stdlib/math/base/special/sqrt.h"
 #include "stdlib/math/base/special/floor.h"
 #include "stdlib/constants/float64/max_safe_integer.h"
+#include "stdlib/math/base/special/fmod.h"
 #include <stdint.h>
 #include <math.h>
```

Comment:
@gunjjoshi Can we now also remove this header file?

Reply:
Ah, yes. Done!